jobs/sql/schemas/eid_visit.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE eid_visit
+(
+    emr_id          VARCHAR(25),
+    encounter_id    INT,
```

Comment:
when you preface encounter_id with partition, you'll need to change this to VARCHAR(25),